src/components/SearchBar.tsx

```diff
@@ -15,14 +15,21 @@ export const SearchBar: React.FC<SearchBarProps> = (props) => {
   }
   return (
     <Container length={length}>
-      <SearchIcon src={process.env.PUBLIC_URL + '/search.svg'} />
-      <SearchInput
+      <img
+        className="search-icon"
+        src={process.env.PUBLIC_URL + '/search.svg'}
+      />
+      <input
+        className="search-input"
         onChange={(e) => setSearchValue(e.target.value)}
         value={searchValue}
       />
-      <ClearButton onClick={handleClearButtonClick}>
-        <ClearButtonIcon src={process.env.PUBLIC_URL + '/clear.svg'} />
-      </ClearButton>
+      <button className="clear-button" onClick={handleClearButtonClick}>
+        <img
+          className="clear-button-icon"
```

Comment:
어디에서쓰이는 clear button인지 알 수 있게 `searchbar-clear-button` 또는 예진님이 위에서 해주셨듯이 `search-clear-button`해주면 좋을거같아요 그냥 파일명을 앞에 먼저붙이면서 클래스네임 써도괜찮을거같슴다
```suggestion
      <button className="search-clear-button" onClick={handleClearButtonClick}>
        <img
          className="search-clear-button-icon"
```